bin/reindex

```diff
@@ -29,11 +30,29 @@ echo "Importing all API content"
 echo "Waiting for empty queues"
 ./bin/wait-for-empty-queues
 
+echo "Check for empty index"
+index_check=`.bin/console index:total:write`
+
+if [ "$index_check" -eq "0" ] || [ "$index_check" -eq "1" ]; then
+
+    # To do: Consider adding additional threshold if items are indexed but
+    # are very different than expected
+    echo "We cannot switch to an index that is empty when performing a reindex"
+
+    echo "Switching writes back to current index $write_index_backup"
+    ./bin/console index:switch:write "$write_index_backup"
+
+    echo "Deleting unused indexes"
+    ./bin/console index:delete:unused
+
+    exit 0
```

Comment:
We should not exit with 0 as this has not been successful and it should reflect that in the jenkins logs.